app/assets/stylesheets/application.scss

```diff
@@ -130,12 +131,20 @@ $icon-size: 0.75rem;
     -ms-transform: scale(1);
     transform: scale(1); }
 
-body {
-  background-color: $jet;
-  background-image: image-url("background.jpg");
-  background-repeat: no-repeat;
-  background-position: center top;
-  background-size: 100%;
+@media only screen and (max-width: 767px) {
+  body {
+    background-color: $jet;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  body {
+    background-color: $jet;
+    background-image: image-url("background.jpg");
+    background-repeat: no-repeat;
```

Comment:
Properties should be ordered background-color, background-image, background-position, background-repeat, background-size